evergreen.yml

```diff
@@ -147,32 +161,110 @@ functions:
+          ./mongoimport.exe --uri="mongodb://$ADF_TEST_LOCAL_HOST:$MDB_TEST_LOCAL_PORT/integration_test" \
+            --drop resources/integration_test/testdata/complex_types.json &&
+          
           
           MONGOSHELL=$(find ./local_adf/ | grep mongo.exe | head -1) &&
           chmod +x $MONGOSHELL &&
           
           # Generate schema on imported data
           $MONGOSHELL -u $ADF_TEST_LOCAL_USER --password $ADF_TEST_LOCAL_PWD --authenticationDatabase \
             $ADF_TEST_LOCAL_AUTH_DB $ADF_TEST_LOCAL_HOST/admin \
-            --eval 'db.runCommand({sqlGenerateSchema: 1, sampleNamespaces: ["supplies.sales"], setSchemas: true})' &&
+            --eval 'db.runCommand({sqlGenerateSchema: 1, 
+            sampleNamespaces: ["integration_test.complex_types", "supplies.sales"], setSchemas: true})' &&
 
-          # SQL-1126: Integration testing
-          # Example command, get collections in supplies
-          $MONGOSHELL -u $ADF_TEST_LOCAL_USER --password $ADF_TEST_LOCAL_PWD --authenticationDatabase \
-            $ADF_TEST_LOCAL_AUTH_DB $ADF_TEST_LOCAL_HOST/supplies  --eval 'db.getCollectionNames()'
+          # Set credentials for the queryFile
+          echo $(eval echo $(cat resources/credentials.json )) | tools/PQTest.exe set-credential \
+            --extension MongoDBAtlasODBC.pqx \
+            --queryFile connector/MongoDBAtlasODBC.integration-test.query.pq &&
+          
+          # Run Integration Test
+          test_output=`tools/PQTest.exe run-test --extension MongoDBAtlasODBC.pqx \
+            --queryFile connector/MongoDBAtlasODBC.integration-test.query.pq`
           EXITCODE=$?
+          failures=`echo $test_output  | grep -Eo '"Result"[^}]*' | grep Failure || true` 
+          echo $test_output > MongoDBAtlasODBC.integration-test.log 
+          if [[ ! -z $failures ]]; then 
+            echo "Failures Encountered:"
+            echo $failures
+            exit 1
+          fi
           
           # Stop local ADF, always run this to keep task from hanging
           ./resources/run_adf.sh stop
           exit $EXITCODE
 
+  "run unit-test":
+    - command: shell.exec
+      params:
+        shell: bash
+        working_dir: mongo-powerbi-connector
+        script: |
+          ${prepare_shell}
+          test_output=`tools/PQTest.exe run-test --extension MongoDBAtlasODBC.pqx \
+          --queryFile connector/MongoDBAtlasODBC.unit-test.query.pq` 
+          failures=`echo $test_output  | grep -Eo '"Result"[^}]*' | grep Failure || true` 
+          echo $test_output > MongoDBAtlasODBC.unit-test.log 
+          if [[ ! -z $failures ]]; then 
+            echo "Failures Encountered:"
+            echo $failures
+            exit 1
+          fi
+
+  "setup driver on Windows":
+    - command: shell.exec
+      params:
+        shell: bash
+        working_dir: mongo-powerbi-connector
+        script: |
+          ${prepare_shell}
+          git clone git@github.com:mongodb/mongo-odbc-driver.git
```

Comment:
I get the odbc repo here and build it.  I'm thinking when we have a release in the odbc driver this can be changed to downloading it instead.  However this would then need to be updated manually.  I'm thinking an env variable would be easier to update, but still require manual intervention.  Let me know what you think.

Reply:
I was thinking about this as well during review. I wonder if we'd be safe with just figuring out the latest release tag (or defaulting to master|main if there isn't one).

Something like 
```sh
TAG="master"
if git describe --tags $(git rev-list --tags --max-count=1); then
  TAG=$(git describe --tags $(git rev-list --tags --max-count=1))
fi
git checkout $TAG
```

Thoughts?

Reply:
ATM, since we have a periodic build (if I don't exclude it from Evergreen on Periodic build)when the release task is merged, the latest build of the ODBC driver (nightly build) will be available from the bucket `translators-connectors-releases` here : mongosql-odbc-driver/windows/test_release/mongoodbc.dll
I was pondering having the release task only opened to tag triggers, but because I wanted to test it out, I checked other project and found that this was not a bad idea.
Let me know what you think. Maybe instead of `test_release`, we could have it named `snapshot` or `nightly`.

Reply:
That's not a bad idea either. I guess it's a question of do we want to build on the latest in repo, or do we want to build on a release?

Reply:
Yes, and I mean, both will be available on the `translators-connectors-releases` bucket. So which one we take is up to us.
@Oliver : If you plan to use the nightly build, let me know. I think it would be best to split the release task in 2. Maybe a `release` and `snapshot` tasks. We don't need to run all unit and integration tests on `release`, but would need for `snapshot` to make sure that it does not simply compile.
I'll hold off merging the tag trigger release PR until I know if you'd like me to make changes.

Reply:
Thank you both for your comments on this.  
@nbagnard There would be value in running on the `snapshot` tasks in that the issues would be caught sooner.  I'll plan on using the nightly build.

Reply:
A bonus is that we can get rid of all the rust build setup. It should speed it up.